WalletWasabi.Backend/Controllers/ArenaController.cs

```diff
@@ -0,0 +1,59 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using WalletWasabi.WabiSabi.Backend.PostRequests;
+using WalletWasabi.WabiSabi.Models;
+
+namespace WalletWasabi.Backend.Controllers
+{
+	/// <summary>
+	/// To make batched requests.
+	/// </summary>
+	[Produces("application/json")]
+	[ApiController]
+	[Route("[controller]")]
+	public class ArenaController : ControllerBase
```

Comment:
does it make sense to merge `ArenaRequestHandler` into this class or is the separation necessary/desirable for a reason i'm missing?

Reply:
The ArenaController is a flawed concept. It leaks implementation details. If someone looks at [the API,](https://wasabiwallet.io/swagger/) this'll lead to WTF moments.

Reply:
Yes, that's right.

Reply:
@nothingmuch controllers are in the backend project and they provide the http interface for the functionality so, they should be separated.